src/main/java/com/iridium/iridiumteams/listeners/InventoryMoveItemListener.java

```diff
@@ -0,0 +1,20 @@
+package com.iridium.iridiumteams.listeners;
+
+import com.iridium.iridiumteams.IridiumTeams;
+import com.iridium.iridiumteams.database.IridiumUser;
+import com.iridium.iridiumteams.database.Team;
+import lombok.AllArgsConstructor;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.inventory.InventoryMoveItemEvent;
+
+@AllArgsConstructor
+public class InventoryMoveItemListener<T extends Team, U extends IridiumUser<T>> implements Listener {
+    private final IridiumTeams<T, U> iridiumTeams;
+    @EventHandler(ignoreCancelled = true)
+    public void onInventoryMoveItem(InventoryMoveItemEvent event) {
+        if (iridiumTeams.getTeamManager().isBankItem(event.getItem())) {
+            event.setCancelled(true);
+        }
+    }
+}
```

Comment:
They should be able to move the item in their inv no?

Reply:
this event is for hoppers

Reply:
its kind of a bad name for it tbh